pontoon/contributors/static/js/settings.js

```diff
@@ -1,7 +1,6 @@
 $(function () {
   function userAttributeURL(action) {
-    const username = $('#profile input[name="username"]').val();
-    return `/user/${username}/attributes/${action}/`;
+    return `/user/attributes/${action}/`;
   }
```

Comment:
With this simplification, this function should be dropped and plain strings used as `url` values.